src/course-unit/add-component/AddComponent.test.jsx

```diff
@@ -338,8 +374,102 @@ describe('<AddComponent />', () => {
     expect(handleCreateNewCourseXBlockMock).toHaveBeenCalled();
     expect(handleCreateNewCourseXBlockMock).toHaveBeenCalledWith({
       parentLocator: '123',
-      category: 'openassessment',
+      category: COMPONENT_TYPES.openassessment,
       boilerplate: 'peer-assessment',
     });
   });
+
+  describe('component support label', () => {
+    it('doesn\'t display component support label if component support legend is disabled', async () => {
```

Comment:
It's advisable to avoid using reverse logic words like 'doesn't display.' It can make the test steps less intuitive to understand. Instead, opting for more affirmative language such as 'is hidden' would enhance clarity.

Let's consider `component support label is hidden if component support legend is disabled`

Reply:
And for other cases the same idea

Reply:
Changed